opm/parser/eclipse/EclipseState/IOConfig/IOConfig.cpp

```diff
@@ -37,6 +38,10 @@ namespace Opm {
         m_ignore_RPTSCHED_RESTART(false){
     }
 
+    bool IOConfig::getWriteInitialRestartFile() const {
```

Comment:
This method should be removed - the required special casing of step 0 should be internal to the `getWriteRestartFile()` method - i.e
```C++
bool IOConfig::getWriteRestartFile(size_t timeStep) {
    if (timeStep == 0)
       return m_write_initial_RST_file;
    else {
       ....
   }
}
```


nstead the `getWriteRestartFile()